core-kotlin/src/test/kotlin/com/baeldung/kotlin/ArrayInitializationTest.kt

```diff
@@ -0,0 +1,48 @@
+package com.baeldung.kotlin
+
+import org.junit.Test
+import kotlin.test.assertEquals
+
+class ArrayInitializationTest {
+
```

Comment:
A few general notes:

1. Please review the code formatting guidelines within **baeldung - for authors - 2 - code in the article** and **baeldung - for authors - 2 - implementation code**. You need to indent your code with spaces instead of tabs and make the use of new lines within your tests consistent with one another and the overall code base.

2. semicolons should not be used in Kotlin -> https://stackoverflow.com/questions/39318457/what-are-the-rules-of-semicolon-inference-in-kotlin

Reply:
I got some problems with formatting because Eclipse Kotlin plugin (as far as I see) doesn't support formatting rules import from XML.  I have formatted the code as Java code. I hope it is ok now.
Also, I made the other requested changes.

Reply:
Changes look great, thanks!

Reply:
Your code changes look good. I will merge them in once we are ready to pass off your article to our senior editor.